initScripts/aarch64__Ubuntu_16.04__Docker_17.06.sh

```diff
@@ -35,6 +35,7 @@ export LEGACY_CI_CEXEC_LOCATION_ON_HOST="/home/shippable/cexec"
 export LEGACY_CI_DOCKER_CLIENT_LATEST="/opt/docker/docker"
 export DEFAULT_TASK_CONTAINER_MOUNTS="-v $BUILD_DIR:$BUILD_DIR \
   -v $REQEXEC_DIR:/reqExec"
+export TASK_CONTAINER_COMMAND="/reqExec/bin/dist/main/main"
```

Comment:
Removed `bash -c` as its unnecessary and creates problems with quote escaping. The binary path is sufficient to start the reqExec.